storage/storage_tests/storage_tests.cpp

```diff
@@ -467,7 +467,7 @@ void OnCountryDownloaded(TCountryId const & countryId, TLocalFilePtr const local
   LOG(LINFO, ("OnCountryDownloaded:", *localFile));
 }
 
-TLocalFilePtr CreateDummyMapFile(CountryFile const & countryFile, int64_t version, size_t size)
+TLocalFilePtr CreateDummyMapFile(CountryFile const & countryFile, int64_t version, uint32_t size)
```

Comment:
Почему не int64_t ?

> On Apr 6, 2016, at 15:43, ygorshenin <notifications@github.com> wrote:
> 
> In storage/storage_tests/storage_tests.cpp <https://github.com/mapsme/omim/pull/2785#discussion_r58697340>:
> 
> > @@ -467,7 +467,7 @@ void OnCountryDownloaded(TCountryId const & countryId, TLocalFilePtr const local
> >    LOG(LINFO, ("OnCountryDownloaded:", *localFile));
> >  }
> >  
> > -TLocalFilePtr CreateDummyMapFile(CountryFile const & countryFile, int64_t version, size_t size)
> > +TLocalFilePtr CreateDummyMapFile(CountryFile const & countryFile, int64_t version, uint32_t size)
> Пусть здесь тоже будет uint64_t
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2785/files/bb6f82e1bd96a8244a5f19432fef96d0dcdfa806#r58697340>